assets/wizards/site-design/views/main/index.js

```diff
@@ -0,0 +1,275 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { alignCenter, alignLeft } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	withWizardScreen,
+	ColorPicker,
+	TextControl,
+	SelectControl,
+	ToggleControl,
+	SectionHeader,
+	ImageUpload,
+	hooks,
+	Button,
+	Grid,
+	WebPreview,
+} from '../../../../components/src';
+import ThemeSelection from '../../components/theme-selection';
+import { getFontsList, getFontImportURL, LOGO_SIZE_OPTIONS, parseLogoSize } from './utils';
+import './style.scss';
+
+const Main = ( {
+	wizardApiFetch,
+	setError,
+	renderPrimaryButton,
+	buttonText,
+	hasPreview = true,
+} ) => {
+	const [ themeSlug, updateThemeSlug ] = useState();
+	const [ mods, updateMods ] = hooks.useObjectState();
+
+	const updateSettings = response => {
+		updateMods( response.theme_mods );
+		updateThemeSlug( response.theme );
+	};
+	useEffect( () => {
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/newspack-setup-wizard/theme',
+		} )
+			.then( updateSettings )
+			.catch( setError );
+	}, [] );
+
+	const saveSettings = () =>
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/newspack-setup-wizard/theme/',
+			method: 'POST',
+			data: { theme_mods: mods, theme: themeSlug },
+			quiet: true,
+		} )
+			.then( updateSettings )
+			.catch( setError );
+
+	return (
+		<>
+			<SectionHeader
+				title={ __( 'Theme', 'newspack' ) }
+				description={ __( 'Activate a theme you like to get started', 'newspack' ) }
+			/>
+			<ThemeSelection theme={ themeSlug } updateTheme={ updateThemeSlug } />
+			<SectionHeader
+				title={ __( 'Colors', 'newspack' ) }
+				description={ __( 'Define your primary and secondary colors', 'newspack' ) }
+			/>
+			<Grid gutter={ 32 }>
+				<ColorPicker
+					label={ __( 'Primary' ) }
+					color={ mods.primary_color_hex }
+					onChange={ updateMods( 'primary_color_hex' ) }
+				/>
+				<ColorPicker
+					label={ __( 'Secondary' ) }
+					color={ mods.secondary_color_hex }
+					onChange={ updateMods( 'secondary_color_hex' ) }
+				/>
+			</Grid>
+			<SectionHeader
+				title={ __( 'Typography', 'newspack' ) }
```

Comment:
(Not a blocker) This is maybe more of a UX/design question than related to code, so maybe for @thomasguillot? I dig that we have the select dropdowns with Google Font options here! Much more user-friendly than entering font name strings. However, we do lose a bit of the flexibility that the Customizer UI provides (being able to use Typekit fonts, for example). Is there a middle ground here?

<img width="459" alt="Screen Shot 2021-03-05 at 2 21 24 PM" src="https://user-images.githubusercontent.com/2230142/110175649-5d271500-7dbf-11eb-8393-282889f76f1f.png">

Also, I love the user-friendliness of the dropdowns, but I found myself wanting to see what each font looks like. We don't need to get fancy with this, but maybe we can provide external links to the Google Fonts pages for selected fonts? e.g. https://fonts.google.com/specimen/Montserrat

A nice enhancement in the future might be to show an actual block of text with a heading and update the font dynamically based on the selections.

Reply:
> I dig that we have the select dropdowns with Google Font options here! Much more user-friendly than entering font name strings. However, we do lose a bit of the flexibility that the Customizer UI provides (being able to use Typekit fonts, for example). Is there a middle ground here?

In my mockup, I actually have a radio option to switch between Curated list and Custom font. But, I think for a v1, it's OK not to have the Custom font option since it's still accessible via the Customizer.

<img width="840" alt="Screenshot 2021-03-09 at 10 07 49" src="https://user-images.githubusercontent.com/177929/110454589-54845680-80bf-11eb-86d1-b52248db4d1d.png">

Worth noting that with Newspack Lite, we will only offer the Curated list (current version).

>  found myself wanting to see what each font looks like. We don't need to get fancy with this, but maybe we can provide external links to the Google Fonts pages for selected fonts?
>
> A nice enhancement in the future might be to show an actual block of text with a heading and update the font dynamically based on the selections.

Here's what I think:

* We already have a "Preview Site" button, so the publication can check the font(s) they selected.
* In a potential v2, I pictured a small iframe with live changes applied to it so it'd be easier for a publication to see the changes
* Having an external link to the font or a random block of text doesn't give proper accuracy when it comes to font combination, layout, and context. I much prefer a "Preview Site" button 😊